stringResources/en-US/resources.resjson

```diff
@@ -37,17 +37,19 @@
 	"Visual_HideStraightLines": "Hide Straight Lines",
 	"Visual_TextColor": "Text Color",
 	"Visual_TicksColor": "Ticks Color",
+	"Visual_Ticks": "Ticks",
 	"Visual_Description_Legend": "Display legend options",
 	"Visual_Description_Title": "Display a title for legend symbols",
 	"Visual_Description_LegendName": "Title text",
 	"Visual_Hightlighted": "Hightlighted",
+	"Visual_Slice": "Slice",
+	"Visual_OnObject_Delete": "Delete",
```

Comment:
lets remove "OnObject" from variables naming if it is not a real part of the result name. Variable should be multifunctional.